app/models/spree/tax/order_adjuster_decorator.rb

```diff
@@ -0,0 +1,14 @@
+module SpreeAvatax
+  module Extensions
+    module OrderAdjuster
+      def adjust!
+        return super unless rates_for_order_zone(order).all?(&:avatax?)
```

Comment:
I think this may be a problem right? This, if I understand, will grab all rates for the order. Then, it is asking if all of these rates respond true for avatax. Should this be `all` or `any`? I have been playing on the [PR app](https://glossier-pr-3151.herokuapp.com) and it doesn't seem to be generating an invoice/sales tax for an order sent to the Lafayette office.

Reply:
I also ran this code in the terminal against the mentioned order (id: 2841345) using the following:
```
tax = Spree::Tax::OrderAdjuster.new(order)
tax.send(:rates_for_order_zone, order).all?(&:avatax?)
  Spree::Calculator Load (1.3ms)  SELECT  "spree_calculators".* FROM "spree_calculators" WHERE "spree_calculators"."calculable_id" = $1 AND "spree_calculators"."calculable_type" = $2 LIMIT 1  [["calculable_id", 1], ["calculable_type", "Spree::TaxRate"]]
=> false

tax.send(:rates_for_order_zone, order).any?(&:avatax?)
  Spree::Calculator Load (1.6ms)  SELECT  "spree_calculators".* FROM "spree_calculators" WHERE "spree_calculators"."calculable_id" = $1 AND "spree_calculators"."calculable_type" = $2 LIMIT 1  [["calculable_id", 2], ["calculable_type", "Spree::TaxRate"]]
  Spree::Calculator Load (1.4ms)  SELECT  "spree_calculators".* FROM "spree_calculators" WHERE "spree_calculators"."calculable_id" = $1 AND "spree_calculators"."calculable_type" = $2 LIMIT 1  [["calculable_id", 3], ["calculable_type", "Spree::TaxRate"]]
=> true
```

Reply:
Hmm. I made the assumption that Avatax would be the only tax-rate covering a given order. Maybe that was a bad assumption?

Reply:
No, it's a legit assumption. No worries there. However, say we focus on just the Lafayette address, this will pull in North America along with Shopify since we have a Shopify popup above the office. This will return both tax rates. The Avatax powered North America one and the none Avatax powered Shopify one.

Reply:
Then I think you're right. Unfortunately, if I go with `any?`, then I have to go through the `ItemAdjuster`. I will make some adjustments and resubmit. :+1: